cmd/docker/docker_test.go

```diff
@@ -30,17 +32,23 @@ var discard = ioutil.NopCloser(bytes.NewBuffer(nil))
 func TestExitStatusForInvalidSubcommandWithHelpFlag(t *testing.T) {
 	cli, err := command.NewDockerCli(command.WithInputStream(discard), command.WithCombinedStreams(ioutil.Discard))
 	assert.NilError(t, err)
-	cmd := newDockerCommand(cli)
-	cmd.SetArgs([]string{"help", "invalid"})
+	tcmd := newDockerCommand(cli)
```

Comment:
Can you factorize these 5 lines in a helper test function (and [flagged](https://golang.org/pkg/testing/#T.Helper) as such)? I think they add some boiling plate code to these pretty succinct tests. (and by the way if you can

Reply:
Probably better to combine some of these tests and rewrite them as subtests (but I'm ok with doing that in a follow up (also see my comment around naming; https://github.com/docker/cli/pull/1722/files/9457f1f070e8fdb6028a5767c78b95d5f09d68d0#r263477744)